code/drasil-example/Drasil/GlassBR/Unitals.hs

```diff
@@ -337,7 +337,7 @@ notSafe       = dcc "notSafe"     (nounPhraseSP "not safe")
 probBreak     = cc prob_br
   ("The fraction of glass lites or plies that would break at the first " ++
     "occurrence of a specified load and duration, typically expressed " ++
-    "in lites per 1000.")
+    "in lites per 1000 [3]." {-astm2016-})
```

Comment:
We really don't want hard-coded references like this `[3]`.

Reply:
We can't really fix this until `Label` is fully implemented, so this is a temporary fix to (a) have the reference there and (b) track what it is with the comment so that it can be actually referenced once we have the appropriate tools.

Reply:
Hmm, alright I guess as long as the hard-coded references are clearly tracked (and not forgotten) it should be okay. I think the citation comment should be a bit more explicit, ie. `FIXME: cite astm2016`